app/controllers/sessions_controller.rb

```diff
@@ -50,9 +50,19 @@ def self.create_password_digest(username, password)
 
   def user_from_params
     login = Session.new(session_params)
-    hash = SessionsController.create_password_digest(login.username,
-                                                     login.password)
-    User.where(name: login.username, password: hash).first
+    user = User.find_by(name: login.username)
+    migrate_to_bcrypt(user, login.password) if user.old_password.present?
```

Comment:
It looks like it will, but just to clarify: this'll let people continue logging in with their existing password without forcing a password reset, correct?